components/SubTrackDetails/SubTrackDetails.jsx

```diff
@@ -0,0 +1,26 @@
+require('./SubTrackDetails.scss')
+
+import React from 'react'
+
+const SubTrackDetails = (props) => {
```

Comment:
Please use destructured properties i.e `(props)` to` {name, code, description, tracks}`